_data/events_gdcr2019/shanghai.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "GDCR 2019 at Ekohe Shanghai",
+  "url": "",
```

Comment:
I'm creating a page on meetup.com. I will update the url field in this PR after the page is created.

Reply:
Hi @rachelcarmena!

I've added the url to the event page. Please check!

Reply:
Awesome, thanks!